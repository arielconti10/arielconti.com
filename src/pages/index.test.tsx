import React from 'react';
import { render, screen } from '@testing-library/react';
import HomePage from './';

// Create a test case for the HomePage component
describe('<HomePage />', () => {
  // Render the component
  it('should render the HomePage component', () => {
    render(<HomePage />);
  });

  // Test the component's snapshot
  it('should match the HomePage component snapshot', () => {
    const { asFragment } = render(<HomePage />);
    expect(asFragment()).toMatchSnapshot();
  });

  // Test the component's content
  it('should render the HomePage component with the correct content', () => {
    render(<HomePage />);
    expect(screen.getByText('Ariel Conti')).toBeInTheDocument();
  });

});
