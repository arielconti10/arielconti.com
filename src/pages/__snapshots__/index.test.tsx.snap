// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should match the HomePage component snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex css-1du3sct"
  >
    <a
      class="text css-a0xwrt"
      href="https://github.com/arielconti10"
      rel="norefer noopener"
      target="_blank"
    >
      @arielconti10
    </a>
  </div>
  <main>
    <div
      class="flex css-3z75l1"
    >
      <div
        class="box css-0"
      >
        <h1
          class="text css-min50y"
        >
          Ariel Conti
        </h1>
        <p
          class="text css-kbn7if"
        >
          Front-end engineer who loves UX, Performance and Design.
        </p>
        <p
          class="text css-kbn7if"
        >
          Brazilian based in Lisboa, Portugal.
        </p>
      </div>
    </div>
  </main>
  <div
    class="flex css-dofn0p"
  >
    Made with ❤️ by
    <a
      class="text css-a0xwrt"
      href="https://github.com/arielconti10"
      rel="norefer noopener"
      target="_blank"
    >
      @arielconti10
    </a>
  </div>
</DocumentFragment>
`;
